### PR TITLE
Update alerting terminology

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

This PR updates the terminology used for alerting. An **alert** is the actual instance of the alert and a **rule** is the set of conditions that are checked periodically to trigger the alert.

**Still to do**: update the APM app docs.

### Related issue

https://github.com/elastic/observability-docs/issues/443

### CODE REVIEW

File: docs/en/observability/metrics-threshold-alert.asciidoc [hunk 1]
```diff
@@ -61,37 +61,46 @@ When configuring an action type, you can define precisely when the alert is trig
...
-- `context.alertState`: Depending on why the action is triggered, this variable resolves to `ALERT`, `NO DATA`, or `ERROR`. `ALERT` means the alert's condition is detected, `NO DATA` means that no data was returned for the time period that the alert queried, and `ERROR` indicates an error when querying the data.
-- `context.reason`: This variable describes why the alert is in its current state. For each of the alert's conditions, it includes the **detected value** of the monitored **metric**, and a description of the **threshold**.
-- `context.timestamp`: This variable resolves to the timestamp of when the alert was evaluated.
+The default message for a metrics threshold rule displays the basic variables you can use.
 
+- `context.group`: This variable resolves to the **group** that the rule conditions detected.
+For Inventory rules, this is the name of a monitored host, pod, container, etc. For metric threshold rules,
```
Comment (dedemorton) [inline docs/en/observability/metrics-threshold-alert.asciidoc:79]:
```suggestion
For Inventory rules, this is the name of a monitored host, pod, container, and so on. For metric threshold rules,
```

**Review by dedemorton — COMMENTED** [review]
A couple of minor comments. Otherwise looks great!

File: docs/en/observability/logs-threshold-alert.asciidoc [hunk 2]
```diff
@@ -38,54 +37,66 @@ A different {es} query type backs each of these comparators, and in some scenari
...
+Similarly, if there was a single group by selected, for example, `host.name`, and Host A matches the conditions, but Host B doesn't,
+one alert is triggered for Host A. If both groups matches the conditions, then two alerts are triggered. 
 
 [IMPORTANT]
 =====
-When group by fields are selected, but no documents are containing the selected field(s) within the given time range (of alert execution), then we can't determine the group(s). This is relevant when the alert condition is sensitive to a certain number of documents, and that number might be 0. To use an example, if we want to know that Host A has less than five documents matching a condition, but the host has stopped reporting logs for the entire duration we were querying, no alert is triggered.
+When group by fields are selected, but no documents contain the selected field(s) within the given time range of when the alert is triggered,
+then you can't determine the group(s). This is relevant when the rule condition is sensitive to a certain number of documents, and
+that number might be `0`. For example, if you want to know that `Host A` has less than five documents matching a condition,
```
[the remark this answers is not in the document]
Comment (EamonnTP) [reply docs/en/observability/logs-threshold-alert.asciidoc:60]: @dedemorton  After reading it again, yeah, it's hard to parse. Maybe this is clearer?

`For example, an alert is not triggered if a host has stopped reporting logs for the duration of the query.`

**Comment by bmorelli25** [comment]
Question: does it make sense to update all of these if some didn't make it into 7.13? I don't know 🤷 . I ask because I'm running BC8 and it looks like only some apps have updated to the new terminology.
Screen Shot 2021-05-19 at 2 25 16 PM

---

**For example, Uptime hasn't:**
<kbd>
Screen Shot 2021-05-19 at 2 24 52 PM
</kbd>

---

**But Metrics has:**
<kbd>
Screen Shot 2021-05-19 at 2 30 33 PM
</kbd>

---

**Even weirder, you select "Create Alert" in a dropdown, only to be taken to a page that says "Create Rule"**
<kbd>
Screen Shot 2021-05-19 at 2 32 45 PM
</kbd>
<kbd>
Screen Shot 2021-05-19 at 2 32 59 PM
</kbd>

**Comment by EamonnTP** [comment]
@bmorelli25 Yeah the terminology does seem a little disjointed, especially for the Settings pages. For the Uptime settings page, I have kept it generic; `configure connectors`.  My understanding is that `Create alert` is correct and consistent across the apps, as you are creating an instance of the alert. The `Create rule` dialog relates to creating a set of conditions that trigger that alert instance. In the Stack Managment app, you still have the heading `Alerts and Insights`, with a list of the rules you created.

**Comment by bmorelli25** [comment]
👍 Got it, thanks!

File: docs/en/observability/configure-uptime-settings.asciidoc [hunk 3]
```diff
@@ -32,34 +32,34 @@ data outside of this pattern.
 image::images/heartbeat-indices.png[Heartbeat indices]
 
 [[configure-uptime-alert-connectors]]
-== Configure alert connectors
+== Configure connectors
```
Comment (bmorelli25) [inline docs/en/observability/configure-uptime-settings.asciidoc:35]: Nice. I think this works well until Uptime migrates away from the deprecated alerting API. There's an issue in Kibana somewhere...

**Review by bmorelli25 — APPROVED** [review]
LGTM

File: docs/en/observability/configure-uptime-settings.asciidoc (hunk 3 above)
Comment (EamonnTP) [reply docs/en/observability/configure-uptime-settings.asciidoc:35]: Thanks for letting me know.
